src/Dimo.Client/Models/PageInfo.cs

```diff
@@ -0,0 +1,14 @@
+namespace Dimo.Client.Models
+{
+    #if NETSTANDARD
+    public class PageInfo
+    {
+        public string EndCursor { get; set; }
+        public bool HasNextPage { get; set; }
+        public bool HasPreviousPage { get; set; }
+        public string StartCursor { get; set; }
+    }
+    #elif NET6_0_OR_GREATER
```

Comment:
if this is a library, do we gain anything by adding net 6.0 specific code?

Reply:
the conditional is intended for `NETSTANDARD`